.babelrc

```diff
@@ -1,4 +1,74 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": ["transform-object-rest-spread"]
+  "presets": [
+    ["es2015", { "modules": false }],
```

Comment:
This PR changes everything to ES6 modules. Now, there's webpack2 that can handle them so there is no need for babel transformation which was main reason why not to use them before.